.travis.yml

```diff
@@ -23,6 +26,10 @@ env:
     - CONTAINER_RUNTIME=docker
     - JOBS=4
 
+os:
+- linux
```

Comment:
I guess this adds an additional target which we already have. If I look here:
https://travis-ci.com/cri-o/cri-o/builds/115294469

I see that we doubled the standard linux runs now with the one: `build, unit test and code coverage report`